sdk/communication/azure-communication-callingserver/azure/communication/callingserver/_callingserver_client.py

```diff
@@ -209,3 +219,38 @@ def join_call(
         )
 
         return CallConnection(join_call_response.call_connection_id, self._call_connection_client)
+
+    @distributed_trace()
+    def start_download(
+        self,
+        content_url,  # type: str
+        **kwargs  # type: Any
+    ):  # type: (...) -> HttpResponse
+        """Start download using content url.
+
+        :param str content_url:
+            The content url.
+        :returns: HttpResponse for a successful download request.
+        :rtype: ~HttpResponse
```

Comment:
@cochi2  Please help review download api.
Should we move start download api to _content_downloader.py?

Reply:
calling server client contains the API which public exposed, it internally calls method from _content_downloader.py